roles/network_validated_content/files/project_config_files/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.black]
```

Comment:
If a validated content isn't supposed to contain plugins (or Python code), is this necessary?